Sources/StreamChatUI/CommonViews/SwiftUIViewRepresentable_Tests.swift

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatUI
+import SwiftUI
+import XCTest
+
+@available(iOS 13.0, *)
+final class SwiftUIViewRepresentable_Tests: XCTestCase {
+    func test_wrappedChatChannelAvatarView() {
+        struct CustomView: View {
+            @EnvironmentObject var uiConfig: UIConfig.ObservableObject
+            let content: (channel: _ChatChannel<NoExtraData>?, currentUserId: UserId?)
+            
+            var body: some View {
+                uiConfig.channelList.itemSubviews.avatarView.asView(content)
+                    .frame(width: 50, height: 50)
+            }
+        }
+        
+        final class CustomAvatarView: ChatChannelAvatarView {
+            override func setUpAppearance() {
+                super.setUpAppearance()
+                
+                presenceAvatarView.avatarView.imageView.backgroundColor = .red
+            }
+        }
+        
+        let channel = ChatChannel.mock(cid: .unique)
+        
+        var config = UIConfig()
+        config.channelList.itemSubviews.avatarView = CustomAvatarView.self
+        let view = CustomView(content: (channel, .unique))
+            .environmentObject(config.asObservableObject)
+        
+        AssertSnapshot(view)
+    }
+}
```

Comment:
tbh I don't think this is needed here. We will probably add snapshot tests for each view.

Reply:
I have already added `SwiftUIRepresentable` conformances to a bunch of views - should I add tests for those then? I felt that adding a snapshot test for each view we add this conformance to might be a little bit unnecessary as what we will essentially test is the wrapper itself over and over again - but maybe it might help unravel some bug in the future

Reply:
Exactly, I'm sure we will find some behavior we won't expect :) Let's test the result not the tooling